css/style.css

```diff
@@ -500,9 +500,29 @@ HEAD .round-2 {
   transform-style: preserve-3d;
 }
 
+
+
+.nav-item:hover hr:nth-child(2){
+    border-bottom: 1px solid #3caea3;
+    display: block;
+    width: 30%;
+    margin-left: 2.65rem;
+    padding-right: 2rem !important; 
+}
+
+
+
+/* ******Team ends ****** */
+
+/************* TEAM **************/
```

Comment:
Just above the teams section is being ended and you are starting it again so instead of starting just keep your styles in the above section itself.